src/iperf_udp.c

```diff
@@ -369,6 +369,7 @@ iperf_udp_accept(struct iperf_test *test)
     socklen_t len;
     int       sz, s;
     int	      rc;
+    char      str[128];
```

Comment:
Kind of nitpicky, but do we want to leave this as 128 or put in a define like WARN_STR_LEN? 128 is pretty common, so I'm okay with just leaving it as is too.

Reply:
New version force-pushed. This uses `WARN_STR_LEN` for these two new warnings. A later PR will fix up the lengths in other places where we use this code idiom.